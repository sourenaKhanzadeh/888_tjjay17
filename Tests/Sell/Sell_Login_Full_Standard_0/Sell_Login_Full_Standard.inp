login 
full-standard
sell
logout
