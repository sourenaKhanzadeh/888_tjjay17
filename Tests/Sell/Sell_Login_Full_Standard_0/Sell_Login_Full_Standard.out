Welcome to the system
Enter session type
Enter operation
Enter the event title you would like to sell
Session terminated
